app/presenters/hotwire_combobox/component.rb

```diff
@@ -204,7 +204,7 @@ def name_when_new
     def name_when_new_on_multiselect_must_match_original_name
       return unless multiselect? && name_when_new.present?
 
-      unless name_when_new.to_s == name
+      unless name_when_new.to_s == hidden_field_name
         errors.add :name_when_new, :must_match_original_name,
           message: "must match the regular name ('#{name}', in this case) on multiselect comboboxes."
```

Comment:
Just realized, would be good to use the hidden field name in the error message as well.

Would appreciate a separate PR if you have time. No worries if not.

Reply:
OK I will make separate PR.